src/devices/Mcp3Adc/Mcp3001.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I'm unsure I like the folder name... Would something like Mcpx00x make sense?

Reply:
i.e. MCP3421 seems to also be ADC (not sure if similar). If there are any exceptions in the future we can figure it out at that point

Reply:
The 3421 is I2C and fairly different in how it is controlled. Perhaps for the future. I will change the folder for the next PR.